src/main/java/sirius/biz/jobs/batch/file/ArchiveImportJob.java

```diff
@@ -104,7 +104,9 @@ protected Optional<ExtractedFile> fetchEntry(String fileName) throws IOException
         Enumeration<? extends ZipEntry> zipEntries = zipFile.entries();
         while (zipEntries.hasMoreElements() && taskContext.isActive()) {
             ZipEntry zipEntry = zipEntries.nextElement();
-
+            if(zipEntry.getName().contains("__MACOSX")) {
```

Comment:
Making it configurable for a list of filenames would be neat.

Also, Windows folders sometimes have icon cache files that may be created in some folders (I don't know the exact logic behind that...)

Reply:
This name comes already in 2 places: https://github.com/search?q=repo%3Ascireum%2Fsirius-biz%20__MACOSX&type=code
In one of them, we also ignore UNIX hidden files (starting with `.`).

We should fork ticket and add a method to sirius.kernel.commons.Files ala `isHidden` and `isMetadata` and use those centrally, like for Windoof `thumbs.db` and others (one task of the ticket would be to compile a list of such files/folders)

Reply:
Or we discuss in Stnadup if we expand the scope of the ticket slightly instead of just throwing the TODO on the ever-growing pile of SIRI issues